android/app/src/main/AndroidManifest.xml

```diff
@@ -8,7 +8,7 @@
          FlutterApplication and put your custom class here. -->
     <application
         android:name="io.flutter.app.FlutterApplication"
-        android:label="nativemoon"
+        android:label="NativeMoon"
```

Comment:
Revert

Reply:
This one is the actual app name, not the package name. Those "package" changes were already reverted

Reply:
Makes no difference for flutter folder shenanigans the.